Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -740,3 +756,32 @@ public enum RestError: Error, CustomStringConvertible {
         }
     }
 }
+
+// URL Session extension
+extension RestRequest: URLSessionDelegate {
+    
+    /// URL session function to allow trusting certain URLs
+    public func urlSession(_ session: URLSession, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+        let method = challenge.protectionSpace.authenticationMethod
+        let host = challenge.protectionSpace.host
+        switch (method, host) {
+        case (NSURLAuthenticationMethodServerTrust, self.url):
+            #if MAC_OS_X_VERSION_10_6
+                let trust = challenge.protectionSpace.serverTrust
+                if let trust = trust {
```

Comment:
@ShihabMehboob Could this be simplified to `if let trust = challenge.protectionSpace.serverTrust {...`?